torch/_jit_internal.py

```diff
@@ -650,26 +651,25 @@ def safe_is_subclass(the_type, super_type):
 
     return optional or union_optional
 
-# fake Python container type for Future/RRef
-T = TypeVar('T')
-
-class Future(Generic[T]):
-    __slots__ = ['__args__']
-
-    def __init__(self, types):
-        self.__args__ = types
-
 def is_future(ann):
     if ann is Future:
-        raise RuntimeError('Attempted to use torch.jit.Future without a '
-                           'contained type. Please add a contained type, e.g. '
-                           'torch.jit.Future[int]')
+        raise RuntimeError(
+            "Attempted to use torch.jit.Future without a "
+            "contained type. Please add a contained type, e.g. "
+            "torch.jit.Future[int]"
```

Comment:
Isn't the `Future` type above deleted? Should this be `torch.futures.Future`?

Reply:
I think because `torch.jit` still imports Future from this file so it still alive. we shouldn't delete it at least I think, because of BC.

Reply:
`torch.jit.Future` and `torch.futures.Future` both work as type class after this PR.

Let me change it to a bare `Future` to avoid showing bias to users.